src/main/java/qna/domain/Question.java

```diff
@@ -88,6 +91,25 @@ public List<Answer> getAnswers() {
         return answers;
     }
 
+    public boolean isDeletableBy(User loginUser) {
+        if (CollectionUtils.isEmpty(answers)) {
+            return true;
+        }
+
+        return answers.stream().allMatch(answer -> answer.isOwner(loginUser));
+    }
+
+    public List<DeleteHistory> delete() {
+        List<DeleteHistory> deleteHistories = deleteAllAnswer();
+        deleted = true;
+        deleteHistories.add(new DeleteHistory(ContentType.QUESTION, getId(), writer, LocalDateTime.now()));
+        return deleteHistories;
+    }
+
+    public List<DeleteHistory> deleteAllAnswer() {
+        return answers.stream().map(Answer::delete).collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    public List<DeleteHistory> deleteAllAnswer() {
        return answers.stream()
                .map(Answer::delete)
                .collect(Collectors.toList());
    }
```

스트림 사용 시 이런 식으로 개행을 하면 가독성이 더 좋아지는 것 같아요! 또한 이 메서드는 현재 클래스 내부에서만 사용되고 있는데, 접근 범위를 더 좁혀도 되지 않을까요?